Tipster/MainWindow.h

```diff
@@ -22,9 +24,14 @@ class MainWindow : public BWindow
 	private:
 		void BuildLayout(void);
 		
-		BView*fTipsterViewContainer;
+		BView* fTipsterViewContainer;
+		BSplitView* fMainSplitView;
 
 		Tipster* fTipsterView;
+		
+		BButton* icon;
+		BString* url;
+		BBitmap* icon_bitmap;
 };
```

Comment:
The above variables should be named fIcon, fURL and fIconBitmap or something alike.